src/pages/en/guides/cms/storyblok.md

```diff
@@ -2,11 +2,188 @@
+          : env.STORYBLOK_PUBLIC_TOKEN,
+      components: {
+        // Add your components here
+      },
+      apiOptions: {
+        // Choose your Storyblok space region 
+        region: 'us',
+      },
+    })
+  ],
+});
+```
+
+The Storyblok integration requires an object with the following properties:
+
+1. `accessToken` - The Storyblok API token you created in the previous step.
+
+    :::tip
+    Since the astro config file does not support environment variables. Use the `loadEnv` function from Vite to load the environment variables.
+    :::
+
+2. `components` - A mapping of Storyblok components to your local components. This is required to render your Storyblok components in your Astro project.
+
+    :::note
+    By default, the Storyblok integration will look for your components in the `src` directory.
+    :::
+
+3. `apiOptions` - An object containing [Storyblok API options](https://github.com/storyblok/storyblok-astro#options). 
+
+    :::caution
+    The default region is `eu`. If you are Storyblok space was created in the US region, you will need to set the `region: 'us'`.
+    :::
+
+### Connecting bloks to Astro components
+
+Create a folder called `storyblok` in your `src` directory. This folder will contain all the Astro components that will match your bloks in your Storyblok blok library.
+
+In this example, we will have a `blogPost` blok content type that will be mapped to a `BlogPost.astro` component.
+
+```ini title="Project Structure"
+├── src/
+│   └── storyblok/
+│       └── BlogPost.astro
+├── .env
+├── astro.config.mjs
+└── package.json
+```
+
+Inside the `BlogPost.astro` component, you can render the content of your blok. You can access the blok content using the `blok` prop.
+
+```astro title="src/storyblok/BlogPost.astro"
+---
+import { storyblokEditable, renderRichText } from '@storyblok/astro'
+
+const { blok } = Astro.props
+const content = renderRichText(blok.body)
+---
+
+<article {...storyblokEditable(blok)}>
+  <h1>{blok.title}</h1>
+  <p>{blok.description}</p>
+  <Fragment set:html={content} />
+</article>
+```
+
+The integration provides utility functions to help you render your content. `storyblokEditable` will enable the Storyblok editor to edit your content and `renderRichText` will render your rich text content to HTML.
+
+Finally add the `BlogPost` component to your Astro config file:
+
+```js title="astro.config.mjs" ins={15}
+import { defineConfig } from 'astro/config';
+import storyblok from '@storyblok/astro';
+import { loadEnv } from 'vite';
+
+const env = loadEnv(import.meta.env.MODE, process.cwd(), 'STORYBLOK');
+
+export default defineConfig({
+  integrations: [
+    storyblok({
+      accessToken:
+        import.meta.env.MODE === 'development'
+          ? env.STORYBLOK_PREVIEW_TOKEN
+          : env.STORYBLOK_PUBLIC_TOKEN,
+      components: {
+        blogPost: 'storyblok/BlogPost',
+      },
+      apiOptions: { 
+        region: 'us',
+      },
+    })
+  ],
+});
+```
+
```

Comment:
I'm hesitant to leave the reader at a point where they can't see anything on the screen (unless that happens early in the "Making a blog" section). What do you think about introducing `<StoryblokComponent />` in this section in the simplest possible way?

```suggestion
To test the setup, you can create a new story with the `blogPost` content type, and fetch and render it directly:

```astro title="pages/test.astro" {8,12}
---
import { useStoryblokApi } from '@storyblok/astro'
import StoryblokComponent from '@storyblok/astro/StoryblokComponent.astro'

const storyblokApi = useStoryblokApi()

const { data } = await storyblokApi.get("cdn/stories/test-post", { version: "draft" });

const content = data.story.content;
---
<StoryblokComponent blok={content} />
```

The `<StoryblokComponent />` references your registered components to render the right one.
```

Reply:
couldn't get the backticks trick to save this 😔

Reply:
I think it's a great idea. I will commit to this to see the difference, thank you Dan!